### PR TITLE
Related to issue #939

### PR DESCRIPTION
**Pull request by rusyasoft**
- FAILURE_RATE_EXCEED exception has been triggered but it has no case in the CircuitBreakerEventDTOFactory
- This issue caused by relation between minimum call numbers and failureRateThreshold

### CODE REVIEW

File: resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/monitoring/endpoint/CircuitBreakerEventDTOFactory.java [hunk 1]
```diff
@@ -54,6 +54,10 @@ public static CircuitBreakerEventDTO createCircuitBreakerEventDTO(CircuitBreaker
                 CircuitBreakerOnCallNotPermittedEvent onCallNotPermittedEvent = (CircuitBreakerOnCallNotPermittedEvent) event;
                 return newCircuitBreakerEventDTOBuilder(onCallNotPermittedEvent)
                     .build();
+            case FAILURE_RATE_EXCEEDED:
```
Comment (dlsrb6342) [inline resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/monitoring/endpoint/CircuitBreakerEventDTOFactory.java]: I think we need to add `SLOW_CALL_RATE_EXCEEDED` type too. 
What do you think @RobWin ?

Comment (RobWin) [reply resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/monitoring/endpoint/CircuitBreakerEventDTOFactory.java]: Yes

Comment (dlsrb6342) [reply resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/monitoring/endpoint/CircuitBreakerEventDTOFactory.java]: @rusyasoft Please add `SLOW_CALL_RATE_EXCEEDED ` too.

Comment (rusyasoft) [reply resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/monitoring/endpoint/CircuitBreakerEventDTOFactory.java]: @dlsrb6342  added SLOW_CALL_RATE_EXCEEDED case

**Pull request merged**